packages/aragon-api/src/index.js

```diff
@@ -547,6 +548,31 @@ export class AppProxy {
     )
   }
 
+  /**
+   *
+   * Trigger an event handler in the application's store
+   *
+   * @param {string} eventName The name of the event to be handled in the reducer
+   * @param {Object} [returnValues={}] Optional event data
+   */
+  trigger (eventName, returnValues = {}) {
+    return this.rpc.send(
+      'trigger',
+      [eventName, returnValues]
+    )
+  }
+
+  /**
+   * subscribe to an observable that emits events created by the frontend event triggers
+   */
+  frontendTriggers () {
```

Comment:
Perhaps we can rename this to just `triggers`, or perhaps `observeTriggers`.

I'm also not sure about thinking about this as a solely `frontend -> background script` type of action; my original thought was to let this be generic and bi-directional. However I hadn't considered how the sending side would ignore its own messages, which might be tricky (and perhaps not worthwhile) to solve.